### PR TITLE
Create Basic Structure of Rockets and display rocket

### PR DESCRIPTION
**Pull request by principlebrothers**
### Complete Milestone 1:
- Create the basic structure of Rocket
-  Fetch Rocket from API
- Display or render rocket on the DOM

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
## Hi @principlebrothers, 

giphy

**You have done a great job 👍 I can see:**

- Following Gitflow ✔️
- Following YAGNI, KISS ☑️
- Following React Concepts (Props, State) 💯
- Commit messages are meaningful 👏


Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Changes are following 🧾

Remove the commented lines of code as noted in the inline comments below.

File: src/components/pages/Nav.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  border-bottom: 2px solid rgb(80, 80, 80);
+  margin-bottom: 1rem;
+  padding: 0.3rem 5rem 0.2rem 3rem;
+}
+
+.logoContainer {
+  display: flex;
+  align-items: center;
+
+  /* background-color: lightgray; */
```
Comment (OyePriscilla) [inline src/components/pages/Nav.css]: Kindly remove this commented line of code to make your work cleaner.

File: src/components/pages/Nav.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.logoIcon {
+  height: 60px;
+  width: 60px;
+}
+
+.navigationLinks {
+  font-size: 24px;
+}
+
+/* NavLink:nth-child(2) {
```
Comment (OyePriscilla) [inline src/components/pages/Nav.css]: Kindly remove this commented lines of code to make your work cleaner.

**Review by OyePriscilla — APPROVED** [review]

File: src/components/pages/Nav.css (hunk 1 above)
Comment (principlebrothers) [reply src/components/pages/Nav.css]: Hi @OyePriscilla, thanks for pointing that out. I will fix it now

File: src/components/pages/Nav.css (hunk 2 above)
Comment (principlebrothers) [reply src/components/pages/Nav.css]: I will do that, please. Thanks for the review

**Pull request merged**